en/firefox/browsers/history/what-is-a-browser.ftl

```diff
@@ -0,0 +1,48 @@
+
+# Variables:
+#   $cookies (url) - link to https://wikipedia.org/wiki/HTTP_cookie
+what-is-a-browser-websites-save = Websites save information about you in files called <a href="{ $cookies }">cookies</a>. They are saved on your computer for the next time you visit that site. Upon your return, the website code will read that file to see that it’s you. For example, when you go to a website and the page remembers your username and password – that’s made possible by a cookie.
+
+what-is-a-browser-there-are-also = There are also cookies that remember more detailed information about you. Perhaps your interests, your web browsing patterns, etc. This means that a site can provide you more targeted content – often in the form of ads. There are types of cookies, called <em>third-party</em> cookies, that come from sites you’re not even visiting at the time and can track you from site to site to gather information about you, which is sometimes sold to other companies. Sometimes you can block these kinds of cookies, though not all browsers allow you to.
+what-is-a-browser-when-you-go-to = When you go to a website and the page remembers your username and password – that’s made possible by a cookie.
+what-is-a-browser-understanding = Understanding privacy
+what-is-a-browser-nearly-all-major = Nearly all major browsers have a private browsing setting. These exist to hide the browsing history from other users on the same computer. Many people think that private browsing or incognito mode will hide both their identity and browsing history from internet service providers, governments and advertisers. They don’t. These settings just clear the history on your system, which is helpful if you’re dealing with sensitive personal information on a shared or public computer. { -brand-name-firefox } goes beyond that.
+what-is-a-browser-firefox-helps = { -brand-name-firefox } helps you be more private online by letting you block trackers from following you around the web.
+what-is-a-browser-making-your-web = Making your web browser work for you
+what-is-a-browser-most-major-web = Most major web browsers let users modify their experience through extensions or add-ons. Extensions are bits of software that you can add to your browser to customize it or add functionality. Extensions can do all kinds of fun and practical things like enabling new features, foreign language dictionaries, or visual appearances and themes.
+what-is-a-browser-all-browser-makers = All browser makers develop their products to display images and video as quickly and smoothly as possible making it easy for you to make the most of the web. They all work hard to make sure users have a browser that is fast, powerful and easy to use. Where they differ is why. It’s important to choose the right browser for you. { -brand-name-mozilla } builds { -brand-name-firefox } to ensure that users have control over their online lives and to ensure that the internet is a global, public resource, accessible to all.
```

Comment:
Add a comma before `making it ...` so it reads `... as quickly and smoothly as possible, making it easy...`